src/main/index.js

```diff
@@ -18,22 +18,48 @@ const isDevelopment = process.env.NODE_ENV !== 'production';
 // global window references prevent them from being garbage-collected
 const _windows = {};
 
-const createWindow = ({search = null, url = 'index.html', ...browserWindowOptions}) => {
-    const window = new BrowserWindow({
-        useContentSize: true,
-        show: false,
-        webPreferences: {
-            nodeIntegration: true
-        },
-        ...browserWindowOptions
-    });
-    const webContents = window.webContents;
+const displayPermissionDeniedWarning = (browserWindow, permissionType) => {
+    let title;
+    let message;
+    switch (permissionType) {
+    case 'camera':
+        title = 'Camera Permission Denied';
+        message = 'Permission to use the camera has been denied. ' +
+            'Scratch will not be able to take a photo or use video sensing blocks.';
```

Comment:
Will these be localized?

Reply:
Currently, NO strings are translated in scratch-desktop. Maybe in next PR?

Reply:
Not yet, but localization is definitely on my list for both scratch-desktop and scratch-link. In terms of "now" or "next" or "later" I would say localization in scratch-desktop is currently filed under "next" :)